codespell_lib/data/dictionary.txt

```diff
@@ -4602,6 +4602,10 @@ coelesce->coalesce
 coercable->coercible
 coerceion->coercion
 coersion->coercion
+coexhist->coexist
+coexhisted->coexisted
+coexhisting->coexisting
+coexhists->coexists
```

Comment:
Can you also separately add the hyphenated versions:
https://dictionary.cambridge.org/dictionary/english/co-exist

Reply:
Added in c1d0d79

Reply:
I also went ahead and did the same for some of the neighbor words as well.